### PR TITLE
 ITS issue fixed for tdnf.

### PR DESCRIPTION
**Pull request by ntsbtz**

### CODE REVIEW

**Comment by ntsbtz** [comment]
@gpiyush-dev please review this .

**Review by gpiyush-dev — COMMENTED** [review]
Not sure if we should modify COPYING file. Rest looks good to me.

**Comment by ntsbtz** [comment]
@tapakund  Is it fine modifying COPYING file? Rest Piyush has reviewed it can you please merge if nothing  required to change.

File: client/api.c [hunk 1]
```diff
@@ -469,7 +469,7 @@ TDNFClean(
 }
 
 //Show count of installed
-//not a dnf command just sanity check
+//not a dnf command just confidence check
```
Comment (sshedi) [inline client/api.c]: I think we can use `// not a tdnf command, simple validation` instead

File: include/tdnf.h [hunk 2]
```diff
@@ -136,7 +136,7 @@ TDNFUpdateInfoSummary(
     PTDNF_UPDATEINFO_SUMMARY* ppSummary
     );
 
-//sanity check. displays current installed count.
+//confidence check. displays current installed count.
```
Comment (sshedi) [inline include/tdnf.h:139]: same here. Use a better comment

**Review by sshedi — CHANGES_REQUESTED** [review]

File: client/api.c (hunk 1 above)
Comment (ntsbtz) [reply client/api.c]: resolved

**Review by keerthanakalyan — APPROVED** [review]

**Pull request merged**